nova/core/roles/caddy/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Including Caddy installation tasks...
+  ansible.builtin.include_tasks: install.yml
+
+- name: Including Caddyfile with variables mode tasks...
+  ansible.builtin.include_tasks: caddyfile_with_vars.yml
+  when: caddy_template_caddyfile_with_vars
+
+- name: Including Caddyfile template tasks...
+  ansible.builtin.include_tasks: caddyfile_with_vars.yml
```

Comment:
I think the file here is `caddy_template.yml`

Reply:
Fixed